packages/terra-divider/tests/wdio/divider-spec.js

```diff
@@ -10,8 +10,29 @@ describe('Divider', () => {
     Terra.should.themeCombinationOfCustomProperties({
       testName: 'themed',
       properties: {
-        '--terra-divider-border-color': 'blue',
-        '--terra-divider-border-top-width': '10px',
```

Comment:
Why was this line removed? It also looks like this screenshot was deleted.

Reply:
Updated: [1deb07c](https://github.com/cerner/terra-core/pull/2296/commits/1deb07c772b96ffb855e2e09c0cff61d7c2d4d00). The screenshot is still there (Default[themed]), but shrank because of the missing style. I added the style back in! It was a victim of an earlier version of the changes.